apps/site/pages/en/blog/events/collab-summit-2024-dublin.md

```diff
@@ -0,0 +1,400 @@
+---
+date: '2024-11-11T00:00:00.000Z'
+category: events
+title: 'Trip report: Node.js collaboration summit (2024 Dublin)'
+layout: blog-post
+author: AugustinMauroy
+---
+
+<!--
+day 1: https://hackmd.io/-8o2Th28QhuNJwO43fafAw
+day 2: https://hackmd.io/V3xtjlcrTIGsemPv8t-tKg
+-->
```

Comment:
Should we keep that ? or we will have somewhere on GitHub. 
cc @joyeecheung

Reply:
i think it's pretty harmless source material to preserve, even if in a comment. i used to during https://github.com/nodejs/nodejs.org/pull/7215/commits/28e3048fcfe4dbe76d81b4667cabe455afd253d9 to resolve a duplication